src/components/navbar/buttons/hamburger.js

```diff
@@ -1,15 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './hamburger.scss';
+import Structure from '../../../structure.json';
+import { IoIosArrowForward } from 'react-icons/io';
+import { IoIosArrowBack } from 'react-icons/io';
+import { GrFormSearch } from 'react-icons/gr';
+
+const Hamburger = ({ showLinks }) => {
+    const [menu] = useState(Structure['menu']);
+    const [showMenu, setShowMenu] = useState(showLinks);
```

Comment:
You don't have to add `const Hamburger = ({ showLinks }) => {`. Instead, just use `const Hamburger = () => {`  change `useState(showLinks);` to `useState(true);`